### PR TITLE
AB#74 tick persistence

### PR DESCRIPTION
**Pull request by michaelschufi**
Working implementation of making a store available to the user, which persists between ticks. The user can now do stuff like this:
```
if (typeof store.timesMovedLeft === 'undefined') {
    store.timesMovedLeft = 0;
}
if (store.timesMovedLeft < 3) {
    move("LEFT");
    store.timesMovedLeft++;
} else {
    move("RIGHT");
}
```
Tests will follow.

### CODE REVIEW

File: game-frontend/src/components/help/HelpText.tsx [hunk 1]
```diff
@@ -11,25 +11,46 @@ if(nearestFood.position.x > userUnit.position.x){
...
       <h2>Hilfe</h2>
-      <p>Ziel des Spiel ist es, am längsten zu Überleben.</p>
-      <h3>API</h3>
+      <p>Ziel des Spiel ist es, am längsten zu überleben.</p>
+      <h3>Spielfiguren</h3>
       <h4>Unit</h4>
       <p>
-        Die <InlineCode>Unit</InlineCode> ist deine Figur, mit welcher du dich
-        ins Abenteuer stürzst.
+        Deine <InlineCode>Unit</InlineCode> (grün markiert) ist deine Figur, mit
```
Comment (StuckiSimon) [inline game-frontend/src/components/help/HelpText.tsx]: `Deine Unit ist deine` 😄

**Review by StuckiSimon — APPROVED** [review]
Looks good to me, in a next iteration we could limit the amount of memory a user has, but this is not part of this task (added an item in the backlog :v:).

File: game-frontend/src/components/help/HelpText.tsx (hunk 1 above)
Comment (michaelschufi) [reply game-frontend/src/components/help/HelpText.tsx]: oops, thanks ^^

**Comment by sir-marc** [comment]
> Tests will follow.

Will they?

**Pull request merged**